src/bootstrap/src/core/build_steps/dist.rs

```diff
@@ -2365,6 +2366,10 @@ impl Step for RustDev {
         // just broadly useful to be able to link against the bundled LLVM.
         tarball.add_dir(builder.llvm_out(target).join("include"), "include");
 
+        tarball.add_dir(builder.gcc_out(target).join("install/lib/libgccjit.so"), "libgccjit.so");
```

Comment:
What will this do on Windows? :thinking: Does the gcc build also work on non-Linux OSes, in general?

Reply:
No it doesn't, good catch!